[Email Routing] Update postmaster.md (#10040)
cloudflare/cloudflare-docs: content/email-routing/postmaster.md

### Subdomains are only supported for Enterprise customers

> Subdomains cannot use Email Routing to forward emails, unless they are part of an Enterprise account. 
> Currently, to achieve this solution you will need to perform the [subdomain setup](https://developers.cloudflare.com/dns/zone-setups/subdomain-setup/) where you can manage the Cloudflare settings for one or more subdomains separately from those associated with your root domain.

Reviewer: ```suggestion
Subdomains cannot use Email Routing to forward emails, unless they are part of an Enterprise account. 
Currently, to achieve this solution you will need to perform a [subdomain setup](https://developers.cloudflare.com/dns/zone-setups/subdomain-setup/). A subdomain setup will allow you to manage the Cloudflare settings for one or more subdomains separately from those associated with your root domain.
```
